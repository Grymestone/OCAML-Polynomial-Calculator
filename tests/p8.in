(8a^5 + 2a^4) * (9a^5 + 1a^1)
