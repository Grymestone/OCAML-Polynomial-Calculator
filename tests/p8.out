(((8*(a)^5)+(2*(a)^4))*((9*(a)^5)+(1*(a)^1)))
18x^9 + 72x^10 + 8x^6 + 2x^5